storage_service/administration/views.py

```diff
@@ -25,6 +26,29 @@ def settings_edit(request):
     return render(request, 'administration/settings_form.html', locals())
 
 
+########################## VERSION ########################
+
+def get_ss_version():
+    try:
+        from storage_service import __version__ as ss_version
+    except:
+        return 'unknown'
+    if not ss_version:
+        return 'unknown'
+    return ss_version
+
+def get_git_commit():
+    try:
+        return subprocess.check_output(['git', 'rev-parse', 'HEAD'])
+    except:
```

Comment:
What exception is this expecting to catch?  It's better to catch the most specific exception expected, so unknown errors don't get by, and we don't accidentally silence exceptions like `KeyboardInterrupt` and `SyntaxError`.  http://stackoverflow.com/questions/21553327/why-is-except-pass-a-bad-programming-practice explains more.